tests/backend-configuration-test.js

```diff
@@ -112,19 +109,19 @@ describeApplication('With valid backend configuration', () => {
     });
 
     it('has a field for the ebsco customer id', () => {
-      expect(SettingsPage.$customerIdField).to.exist;
+      expect(SettingsPage.customerId).to.not.equal('');
     });
 
     it('has a field for the ebsco RM API key', () => {
-      expect(SettingsPage.$apiKeyField).to.exist;
+      expect(SettingsPage.apiKey).to.not.equal('');
     });
 
     it('field for the ebsco RM API key appears with text as password hidden', () => {
-      expect(SettingsPage.$apiKeyField).to.have.prop('type', 'password');
+      expect(SettingsPage.apiKeyInputType).to.equal('password');
     });
 
-    it.still('does not have visible form action buttons', () => {
-      expect(SettingsPage.$actions).to.not.exist;
+    it('does not have visible form action buttons', () => {
```

Comment:
This should be `always`, but that flakiness we found before means it might have to be skipped 😞